pyproject.toml

```diff
@@ -0,0 +1,91 @@
+[tool.poetry]
+name = "hyperstyle-analysis-prod"
+version = "0.1.0"
+description = "Repository containing several tools-helpers to work with data from the Hyperstyle platform"
+authors = ["Anastasia Birillo <anastasia.birillo@jetbrains.com>"]
+readme = "README.md"
+packages = [
+    { include = "core" },
+    { include = "data_collection" },
+    { include = "data_labelling" },
+    { include = "jba" },
+    { include = "preprocessing" },
+    { include = "templates" },
+]
+
+# Core dependencies
+[tool.poetry.dependencies]
+python = "~3.9"
```

Comment:
I'd suggest to use a hard version of Python to be sure about the same environment everywhere

Reply:
Poetry cannot install the specified Python version, so you must do it manually. Therefore I used the tilde requirement here